### DISCUSSION TITLE
Finding latest compatible expo library version for current SDK version

### DISCUSSION THREAD

**Discussion by jarredt** [expo/expo#15895] [Archived: Questions]
Is there a resource that shows what are the max compatible Expo library versions for a given Expo SDK version at any given time?

We've enabled `dependabot` on our app's repository, and it often prompts us to update to the latest version of an `expo-*`library -- but I can't seem to easily figure out if that version is compatible with our SDK version or not.

For context -- in our app, we pin our dependencies to specific versions (i.e. `"expo-cellular": "4.0.0"`) and don't use `^` or `~` -- this is so we can be intentional about our upgrades and ensure we're not bitten by breaking changes in minor/patch updates.

**Comment by raajnadar** [comment]
Run the `expo upgrade <sdk-version>` to get packages as per the current sdk.

When installing new package use `expo install <package-name>`

**Comment by jerone** [comment]
Here is a JSON file containing all Expo libraries: https://github.com/expo/expo/blob/main/packages/expo/bundledNativeModules.json
